standardfiles/orb/.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @sous-chefs/orbs
```

Comment:
Is this user on every repo? It will need to be added for this to work.

Reply:
Its a group and for all orbs repos (circleci) that should be the group anyway